services/brig/src/Brig/Team/API.hs

```diff
@@ -196,9 +211,43 @@ getInvitationByCode (_ ::: c) = do
     inv <- lift $ DB.lookupInvitationByCode c
     maybe (throwStd invalidInvitationCode) (return . json) inv
 
+suspendTeam :: JSON ::: TeamId -> Handler Response
+suspendTeam (_ ::: tid) = do
+    changeTeamAccountStatuses tid Suspended
+    rs <- lift $ DB.lookupInvitations tid Nothing (unsafeRange 100)
+    deleteInvitations rs
+    return empty
+  where
+    deleteInvitations :: DB.ResultPage Invitation -> Handler ()
+    deleteInvitations page = do
+        e <- ask
+        let iids = inInvitation <$> DB.resultList page
+        void $ liftIO $ mapConcurrently (del e) iids
+        when (DB.resultHasMore page) $
+            lift (DB.lookupInvitations tid (Just $ last iids) (unsafeRange 100)) >>= deleteInvitations
```

Comment:
`last` could eventually raise an exception if the list is empty; we can just paginate using `nextPage` like is done [here](https://github.com/wireapp/wire-server/blob/develop/services/brig/src/Brig/Data/Connection.hs#L107-L117)